app/javascript/modules/listings/SharedHelpers.tsx

```diff
@@ -65,8 +65,12 @@ export const getListingAddressString = (listing: RailsListing) => {
     (listing.Building_Street_Address &&
       listing.Building_City &&
       listing.Building_State &&
-      listing.Building_Zip_Code &&
-      `${listing.Building_Street_Address}, ${listing.Building_City} ${listing.Building_State}, ${listing.Building_Zip_Code}`) ??
+      listing.Building_Zip_Code && (
+        <span>
+          {listing.Building_Street_Address}, {listing.Building_City}{" "}
+          <abbr title="California">{listing.Building_State}</abbr>, {listing.Building_Zip_Code}
```

Comment:
Feels odd to hard code the abbreviation `title` and not the state code. Sounds like we won't ever get a non Cali listing, but i would recommend we either hard code both or neither. So, one option could be to add constants for all the state codes and look up the CA value. I'm not sure if we currently have a scenario where we'd use this elsewhere. Perhaps someone in an applicant's household could have an out of state address.